doc/tutorial/basic/tutorial.rst

```diff
@@ -136,11 +136,43 @@ learn::
     <sphx_glr_auto_examples_classification_plot_digits_classification.py>` illustrates how starting
     from the original problem one can shape the data for consumption in
     scikit-learn.
-    
+
 .. topic:: Loading from external datasets
 
     To load from an external dataset, please refer to :ref:`loading external datasets <external_datasets>`.
 
+.. topic:: Loading from the data files
```

Comment:
I don't get why this belongs in the tutorial, unless it's framed as "You can also load your own data. For example, load_boston(...) just pulls in data using numpy.loadtxt::". This currently appears to be too much detail on the internals of scikit-learn.

Reply:
@jnothman, I agree with you, no need to mention the `filename` attribute in the tutorial. Too much detail for a tutorial. 
Should I remove this section then?